tasks/views.py

```diff
@@ -16,31 +16,37 @@ def view_tasks(request):
 
 
 def new_task(request):
-    if request.method == "POST":
-        form = TaskForm(request.user.id, request.POST)
-        taskForm = form.save(commit=False)
-        if form.is_valid():
-            try:
-                Task.create_task(taskForm.title, taskForm.created_by, taskForm.assignee, taskForm.priority,
-                                 taskForm.status, taskForm.description)
-
-                messages.success(request, 'Task was added successfully')
+    context = {}
+    if request.user.is_authenticated:
+        logged_user = User.objects.get(user=request.user)
+        context = {'user': logged_user}
+        if logged_user.is_manager():
+            color = 'red'
+            if request.method == "POST":
+                form = TaskForm(request.user.id, request.POST)
+                task_form = form.save(commit=False)
+                initial_dict = {'title': task_form.title, 'assignee': task_form.assignee,
+                                'priority': task_form.priority, 'status': task_form.status,
+                                'description': task_form.description}
+                if form.is_valid():
+                    try:
+                        Task.create_task(task_form.title, task_form.assignee, task_form.created_by, task_form.priority,
+                                         task_form.status, task_form.description)
+                        messages.success(request, 'Task was added successfully')
+                        color = 'green'
+                        form = TaskForm(request.user.id)
+                    except Exception as e:
+                        messages.warning(request, e)
+                        form = TaskForm(request.user.id,
+                                        initial=initial_dict)
```

Comment:
please do it in one line:
```suggestion
                        form = TaskForm(request.user.id, initial=initial_dict)
```